README.md

```diff
@@ -100,6 +100,21 @@ _Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires yo
 
 ---
 </details>
+<details>
+<summary>Via <strong>Mint</strong></summary>
+
+To install SwiftGen via [Mint](https://github.com/yonaskolb/Mint), simply use:
+
+Note: Will work for Releases > 5.3.0 only.
+
+```sh
+$ brew install libxml2
+$ mint install SwiftGen/SwiftGen --global=false
```

Comment:
I don't think we need the `--global=false`.

I also just checked and this isn't even working in the latest release of mint...
It has been replaced by an opposite `--prevent-global` or `-p`

Reply:
better?

Reply:
That technically works, but it probably doesn't need to be in the install instructions at all. We don't want to assume how people are installing it

Reply:
than we should perhaps add "(system-wide installation)" (see homebrew)